app/views/pages/adopt.html.erb

```diff
@@ -1,7 +1,28 @@
 <div class="stats">
   <ul>
-    <li>"Rats available"<span>Rats available</span></li>
+    <li><%= @available_rats.count %><span>Rats available</span></li>
     <li>"Adopted"<span>Adopted</span></li>
     <li>"Adopted last month"<span>Adopted last month</span></li>
   </ul>
 </div>
+
+<div class="text">
+  <h2>
+    <%= "Rats looking for their forever home." %>
+  </h2>
+  <%= "Mouse over their picture for their name. Click the image for more information." %>
+</div>
+
+<div>
+  <ul class="list-of-rats">
+    <% @available_rats.each do |rat| %>
+      <li>
+        <%= link_to("http://www.petfinder.com/petdetail/#{rat.id}") do %>
+        <%= image_tag rat.photos.first.medium, class: "rat-photo" %>
```

Comment:
Typically when a block closure has been opened like you've done here with the `do` and `end` you'll want to indent the lines between them. This helps convey what is part of that block and what is not. What would you think about indenting this line and the line below?